dev/puppet-server.conf.sample

```diff
@@ -30,8 +30,11 @@ webserver: {
 # configure the mount points for the web apps
 web-router-service: {
     "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
-    "puppetlabs.services.master.master-service/master-service": "/puppet"
+    "puppetlabs.services.master.master-service/master-service": {master-routes:       "/puppet"
 
+                                                                 # The following controls 404 errors and should not
+                                                                 # be modified
+                                                                 invalid-in-puppet-4: "/" }
```

Comment:
@jeffmccune I believe @nfagerlund raised that concern earlier, and @cprice404 addressed it above. Basically, only handling known URLs would cause us to have a number of problems that we're currently trying to get away from having around camping out on the "/" URL namespace since the environment could be anything, so we've decided that the catch-all 404 handler is the right approach for now.